pkg/operator/configobservation/images/observe_images.go

```diff
@@ -30,14 +31,18 @@ func ObserveInternalRegistryHostname(genericListers configobserver.Listers, exis
 	observedConfig := map[string]interface{}{}
 	configImage, err := listers.ImageConfigLister.Get("cluster")
 	if errors.IsNotFound(err) {
-		glog.Warningf("image.config.openshift.io/cluster: not found")
+		glog.V(4).Infof("image.config.openshift.io/cluster: not found")
 		return observedConfig, errs
 	}
 	if err != nil {
+		glog.Warningf("error getting image.config.openshift.io/cluster: %v", err)
 		return prevObservedConfig, errs
 	}
 	internalRegistryHostName := configImage.Status.InternalRegistryHostname
 	if len(internalRegistryHostName) > 0 {
+		if currentInternalRegistryHostname != internalRegistryHostName {
+			glog.V(4).Infof("setting internal registry hostname to: %q", internalRegistryHostName)
```

Comment:
I expect this to be done in the library-go method.  Perhaps you want a better diff here: https://github.com/openshift/library-go/blob/master/pkg/operator/configobserver/config_observer_controller.go#L107 ?